rest/admin_api.go

```diff
@@ -621,7 +628,8 @@ func (h *handler) handleGetDbConfigSync() error {
 		}
 	}
 
-	h.response.Header().Set("ETag", etagVersion)
+	quoteETag := strconv.Quote(etagVersion)
+	h.response.Header().Set("ETag", quoteETag)
```

Comment:
It may make sense to pull the logic out into a common `h.setEtag(etagVersion)` function so all handlers don't have to know the intricacies of quoting Etag values.